packages/next-ui/Config.graphqls

```diff
@@ -0,0 +1,6 @@
+extend input GraphCommerceConfig {
+  """
+  Hide the back button
+  """
+  hideBackButton: Boolean
```

Comment:
This sounds like it could just be a prop on `<LayoutHeaderBack/>` and passed through from `<LayoutHeader/>`?

Reply:
LayoutHeader is also a next-ui component, so that would mean you'd have to pass it to LayoutHeader on every single page individually.

Reply:
Might it be an idea to:
- Create a prop to be able to hide the back button on LayoutHeader or something
- Create a local plugin